ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -112,6 +115,16 @@ const styles = reactStyles({
 
 const pageId = 'workspace';
 
+const PublishButtonToolTipText = (isWorkspaceOwner: boolean) => {
```

Comment:
by convention, initial capital letters are reserved for functions which are used **as React components**.  It's a little tricky because it's context-dependent.  Here, we're using this **as a function** so lowercase is appropriate.

If instead this was accessed like `<PublishButtonToolTipText\>` then initial-capital would be appropriate.